frontend/public/locales/en/translation.json

```diff
@@ -2309,7 +2309,6 @@
     "template.modal.linkProvider.submit": "Link",
     "template.modal.linkProvider.submitting": "Linking",
     "template.modal.linkProvider.title": "Link an Ansible connection",
-    "template.modal.name.helper.text": "The Ansible job template name must match the name of the job on the Ansible Tower. If the names are not the same, the hook cannot run the job.",
```

Comment:
The user cannot enter their own name here - if it doesn't match one we fetched from the server, it is cleared. So this text does not make sense.